doc/devdocs/reflection.rst

```diff
@@ -106,10 +106,7 @@ variable assignments:
    julia> expand( :(f() = 1) )
    :(begin
            $(Expr(:method, :f))
-           $(Expr(:method, :f, :((Core.svec)((Core.apply_type)(Tuple,(Core.Typeof)(f)),(Core.svec)())), Toplevel LambdaInfo thunk
-   :(begin  # none, line 1:
-           return 1
-       end), false))
+           $(Expr(:method, :f, :((Core.svec)((Core.apply_type)(Tuple,(Core.Typeof)(f)),(Core.svec)())), Toplevel LambdaInfo thunk, false))
```

Comment:
This is basically the same issue as https://github.com/JuliaLang/julia/pull/17636 and is kind of annoy.